python/ray/tune/logger.py

```diff
@@ -175,13 +172,18 @@ def _init(self):
         self._csv_out = None
 
     def on_result(self, result):
+        tmp = result.copy()
+        if "config" in tmp:
+            del tmp["config"]
+        result = flatten_dict(tmp, delimiter="/")
```

Comment:
cc @hartikainen this may affect viskit visualization?

Reply:
I think this should be fine. In softlearning, I actually flatten all the dicts in a similar way before returning the results.